app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -185,8 +185,10 @@ public void setBlockingEnabled(boolean enabled) {
                 applyAppSettings();
             } else {
                 if (geckoSession != null) {
+                    geckoRuntime.getSettings().setTrackingProtectionCategories(0);
                     geckoRuntime.getSettings().setJavaScriptEnabled(true);
                     geckoRuntime.getSettings().setWebFontsEnabled(true);
+                    geckoRuntime.getSettings().setCookieBehavior(GeckoRuntimeSettings.COOKIE_ACCEPT_ALL);
```

Comment:
Doesn't this disable all those features globally? Will we re-enable them if a user switches tabs?

Reply:
Yes we reenable if a user switches tabs, setBlockingEnabled is part of tab initialization